### PR TITLE
Changed amount of offers by role

### PR DESCRIPTION
**Pull request by Yurenko**
image
image

### CODE REVIEW

**Review by tsivkadi — APPROVED** [review]

File: src/components/popular-categories/PopularCategories.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ import serviceIcon from '~/assets/img/student-home-page/service_icon.png'
 import CardsList from '~/components/cards-list/CardsList'
 import { CategoryInterface } from '~/types'
 import useBreakpoints from '~/hooks/use-breakpoints'
```
Comment (Tolik170) [inline src/components/popular-categories/PopularCategories.tsx:19]: add space after this line

File: src/components/popular-categories/PopularCategories.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import Box from '@mui/material/Box'
 
 import useAxios from '~/hooks/use-axios'
 import { categoryService } from '~/services/category-service'
+import { useAppSelector } from '~/hooks/use-redux'
```
Comment (Tolik170) [inline src/components/popular-categories/PopularCategories.tsx:10]: remove this space

File: src/pages/subjects/Subjects.tsx [hunk 3]
```diff
@@ -7,34 +7,36 @@ import Container from '@mui/material/Container'
...
-import { categoryService } from '~/services/category-service'
-import { authRoutes } from '~/router/constants/authRoutes'
-import {
-  CategoryNameInterface,
-  SizeEnum,
-  SubjectInterface,
-  SubjectNameInterface
-} from '~/types'
-import { itemsLoadLimit } from '~/constants'
 import serviceIcon from '~/assets/img/student-home-page/service_icon.png'
```
Comment (Tolik170) [inline src/pages/subjects/Subjects.tsx:22]: organize imports

**Review by Tolik170 — APPROVED** [review]

**Review by FryDay969 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

File: src/components/popular-categories/PopularCategories.tsx (hunk 1 above)
Comment (Yurenko) [reply src/components/popular-categories/PopularCategories.tsx:18]: fixed

File: src/components/popular-categories/PopularCategories.tsx (hunk 2 above)
Comment (Yurenko) [reply src/components/popular-categories/PopularCategories.tsx]: fixed

**Pull request merged**